code/client/src/App.js

```diff
@@ -32,6 +32,14 @@ import Reset_token from './components/Pages/Reset_token';
 import ReactDOM from 'react-dom'; 
 
 
+fetch("../../data/country_electricity")
+	.then(function(resp) {
+		return resp.json();
+	})
+	.then(function(data) {
+		console.log(data);
+	})
+
```

Comment:
What is this doing?

Reply:
Thanks for pointing this out, I was trying to display the json file to test if it worked. I'll delete and push!